third_party/move/move-compiler-v2/transactional-tests/tests/inlining/bug_10991_noparam.exp

```diff
@@ -0,0 +1,67 @@
+processed 2 tasks
+
+task 0 'publish'. lines 1-10:
+
+
+
+==> Compiler v2 delivered same results!
```

Comment:
Should we keep `--print-bytecode` on once we have debugged? There is a chance to get some unrelated baseline errors if we update the codegen.